packages/ckeditor5-table/tests/tablecolumnresize/utils.js

```diff
@@ -389,6 +390,8 @@ describe( 'TableColumnResize utils', () => {
 			const row0 = [ ...table.getChildren() ][ 0 ];
 			const cell00 = [ ...row0.getChildren() ][ 0 ];
 
+			tableColumnResizeMouseSimulator.down( editor, getDomResizer( getDomTable( editor.editing.view ), 1, 0 ) );
```

Comment:
Let's extract `getDomResizer( getDomTable( editor.editing.view ), 1, 0 )` call to some helper.

Reply:
It makes sense to leave a single line comment why this call is needed. To my understanding it's due to lazy calculations.

Reply:
It doesn't affect tests, so I removed this line instead. I think it's due to the refactor done in the editing plugin.